README.md

```diff
@@ -22,12 +22,12 @@ It's used in the [snap framework](http://github.com/intelsdi-x/snap).
 
 ### System Requirements
 
-* [golang 1.5+](https://golang.org/dl/) (needed only for building)
+* [golang 1.6+](https://golang.org/dl/) (needed only for building)
 
 ### Installation
 
 #### Download File plugin binary:
-You can get the pre-built binaries for your OS and architecture at snap's [GitHub Releases](https://github.com/intelsdi-x/snap/releases) page.
+You can get the pre-built binaries for your OS and architecture at Snap's [GitHub Releases](https://github.com/intelsdi-x/snap/releases) page.
```

Comment:
This should be: https://github.com/intelsdi-x/snap-plugin-publisher-file/releases, not the snap/release page.